docs/getting-started/agent-tamper-protection.asciidoc

```diff
@@ -0,0 +1,47 @@
+[[agent-tamper-protection]]
+= Prevent {agent} uninstallation
+
+For hosts enrolled in {elastic-defend}, you can prevent unauthorized attempts to uninstall {agent} and {elastic-endpoint} by enabling *Agent tamper protection* on the Agent policy. This offers an additional layer of security by preventing users from bypassing or disabling {elastic-defend}'s endpoint protections. 
+
+When enabled, {agent} and {elastic-endpoint} can only be uninstalled on the host by including an uninstall token in the uninstall CLI command. One unique uninstall token is generated per Agent policy, and you can retrieve uninstall tokens in an Agent policy's settings or in the {fleet} UI.
+
+.Requirements
+[sidebar]
+--
+* Agent tamper protection requires a https://www.elastic.co/pricing[Platinum or higher subscription]. 
+
+* Hosts must be enrolled in the {elastic-defend} integration.
+
+* {agent}s must be version 8.11.0 or later to use this feature.
```

Comment:
Only suggesting this because it's already implied that these are pre-reqs for using the described feature.

```suggestion
* {agent}s must be version 8.11.0 or later.
```